Editor/EditorUtilities/EditorWebRequest.cs

```diff
@@ -67,12 +93,24 @@ public void Send(Action<UnityWebRequest> callback)
         /// </summary>
         public string DataAsText => UnityRequest.downloadHandler.text;
 
+        private float m_LastUpdate;
+        
         void OnEditorUpdate()
         {
             if (m_ShowProgress)
             {
-                var progress = $"Download Progress: {Convert.ToInt32(UnityRequest.downloadProgress * 100f)}%";
-                EditorUtility.DisplayProgressBar(m_ProgressDialogTitle, progress, UnityRequest.downloadProgress);
+                if (!Application.isBatchMode)
+                {
+                    if (Time.realtimeSinceStartup > m_LastUpdate + 1)
+                    {
+                        Debug.Log(HandlerProgress);

```

Comment:
I don't think this is a good idea to spam console. 
If you need this - maybe we can have kind of a "debug mode".

Reply:
In batchmode we don't have anything except console. If file is large, for ex. 1Gb, and network is slow we will never know if something is happening.